examples/testdata/wasi_args.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+)
+
+//go:wasm-module wasi_snapshot_preview1
+//export args_get
```

Comment:
I'm a little confused because I would have expected a test to import args_get, not export it. can you comment why?

Reply:
`export` is TinyGo's syntax to specify its the link-time name, not only for exporting a function. This makes TinyGo import external functions from other wasm modules. https://github.com/tinygo-org/tinygo/issues/1120 https://github.com/tinygo-org/tinygo/blob/06df31944c6af706d6cb8b30a2e92d19060436d2/src/runtime/runtime_wasm_wasi.go#L115-L117